app/AppCache.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+require_once __DIR__.'/AppKernel.php';
+
+use Symfony\Bundle\FrameworkBundle\HttpCache\HttpCache;
+
+class AppCache extends HttpCache
+{
+
+
+
+var $hogeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee='fugaaaaaa';
```

Comment:
__[development][code_sniffer]__
* Line indented incorrectly; expected at least 4 spaces, found 0
* The var keyword must not be used to declare a property
* Visibility must be declared on property "$hogeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee"
* Scope modifier not specified for member variable "$hogeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee"
* Add a single space around assignment operators